src/Serilog.Enrichers.ClientInfo/Enrichers/ClientIpEnricher.cs

```diff
@@ -34,22 +35,9 @@ public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
             return;
         }
 
-        if (httpContext.Items[IpAddressItemKey] is LogEventProperty logEventProperty)
-        {
-            logEvent.AddPropertyIfAbsent(logEventProperty);
-            return;
-        }
-
-        var ipAddress = _contextAccessor.HttpContext?.Connection?.RemoteIpAddress?.ToString();
-
-        if (string.IsNullOrWhiteSpace(ipAddress))
-        {
-            ipAddress = "unknown";
-        }
-
-        var ipAddressProperty = new LogEventProperty(IpAddressPropertyName, new ScalarValue(ipAddress));
-        httpContext.Items.Add(IpAddressItemKey, ipAddressProperty);
+        var ipAddress = httpContext.Connection.RemoteIpAddress?.ToString() ?? "unknown";
```

Comment:
Maybe it makes sense to discard the `unknown` constant altogether and not add this `LogEventProperty` if we don’t have `RemoteIpAddress`? Something like this:
``` csharp
    public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
    {
        var ipAddress = _contextAccessor.HttpContext?.Connection.RemoteIpAddress?.ToString();
        if (string.IsNullOrEmpty(ipAddress))
        {
            return;
        }

        LogEventProperty ipAddressProperty = new(IpAddressPropertyName, new ScalarValue(ipAddress));
        logEvent.AddPropertyIfAbsent(ipAddressProperty);
    }
````
What do you think?

Reply:
I can remove it, but the issue is that a developer might not be able to determine whether they have configured the enricher correctly.

Reply:
Ok, since the address may be empty in case a non-IP transport is used or in a test environment - and this constant may help developers, let's keep it.